src/VkBootstrap.h

```diff
@@ -844,6 +844,13 @@ class SwapchainBuilder {
 	// Set the number of views in for multiview/stereo surface
 	SwapchainBuilder& set_image_array_layer_count(uint32_t array_layer_count);
 
+	// Sets the desired minimum image count for the swapchain. Note that the presentation engine is always free to create more images than requested.
+	SwapchainBuilder& set_desired_min_image_count(uint32_t min_image_count);
+	// Use the default desired minimum image count for the swapchain. The chosen value is calculated as `capabilitites.minImageCount + add_to_min_image_count`.
+	// The default behavior so far was equivalent to setting `add_to_min_image_count` to 1, and generally led to a triple buffering setup.
+	// A discussion of the trade-offs involved between double buffering and triple buffering is available as https://en.wikipedia.org/wiki/Multiple_buffering
```

Comment:
Not sure this is the place to link to a 'general' discussion of multiple buffering, especially since this is in the context of vulkan. If there was a more vulkan focused link I'd be all for it.

Reply:
Yes, I wasn't too sure about it either; I haven't searched for Vulkan-specific resources because the concepts apply to any API.
I don't mind removing it.